docs/API.md

```diff
@@ -242,6 +249,24 @@ The logging level is a *minimum* level. For instance if `logger.level` is
 
 You can pass `'silent'` to disable logging.
 
+It is possible to pass an object to set different levels to the logger and its
+children. The syntax for designating the target loggers is borrowed from the
+[debug](https://github.com/visionmedia/debug#wildcards) module's `DEBUG` variable
+(also supported in [pino-debug](https://github.com/pinojs/pino-debug)):
++ use `'*'`(star) as wildcard (rules are ordered so that `'*'` will not override
+`'mylib*'`).
```

Comment:
I'm fine with referencing the `debug` docs. I just want to get a full understanding from  Pino's own docs.